modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
 
-* Node configuration changes are reflected in newer versions of the installer scripts, and are also delivered via the `rtfctl` command-line utility. These changes include adjusting operating system limits and performing jobs to clean up system resources.
+* The `rtfctl` command-line utility, which is used to manage Runtime Fabric and perform the actual upgrades to cluster-level and node configuration components.
++
+Upgrades are delivered as a new binary via the *Runtime Fabric downloads* page in Runtime Manager.
 
-* The `rtfctl` command-line utility upgrades are provided as a new binary that replaces the existing binary. The `rtfctl` utility is used to manage Runtime Fabric locally.
+* Node configuration, which includes scripts for provisioning AWS and Azure infrastructure, preparing each node, bootstrapping a new Runtime Fabric, adjusting operating system limits, and performing jobs to clean up system resources.
++
+Changes are delivered as an updated install script version via the *Runtime Fabric downloads* page in Runtime Manager.
++
+[NOTE]
+If you are upgrading the node configuration with install scripts, the latest version of the cluster-level components (installer package) is automatically downloaded and used.
 
-Upgrades to cluster-level and node configuration components are performed using the `rtfctl` command-line utility.
+It is important to run with the most current versions of the cluster-level (installer package) and `rtfctl` components.
 
-## Downloads page
+== Determine the Component Versions You Are Using
 
+* Run the following command:
++
+```
+/opt/anypoint/runtimefabric/rtfctl version
+```
+* The output provides the `rtfctl` version, the installer package version in the `appliance` field, and the Runtime Fabric version in the `agent` field, as shown in the following example:
++
+```
+[root@ip-10-0-245-74 /]# /opt/anypoint/runtimefabric/rtfctl version
+COMPONENT    VERSION                  
+rtfctl       0.2.96+2868e14           
+agent        1.5.4                    
+appliance    1.1.1581641679-5884bce   
+kubernetes   1.13.12
+```
++
+[NOTE]
+The Runtime Fabric version is also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, "Upgrade to vx.x" is displayed.
+* Determine the install script version you are using by ???????????
+
+== Determine the Latest Available Component Versions
 The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
 
-The *Runtime Fabric downloads* page also provides an installer link.
+You can also review the xref:release-notes/runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+
+== Runtime Fabric Component Upgrade Considerations
```

Comment:
I think it makes more sense to show the considerations, prerequisites and procedure together for each component upgrade.

Having it arranged in a way where we group the considerations and procedure doesn't seem to flow if we consider the use case of a user trying to figure out "how do I upgrade my appliance version."

Reply:
I've consolidated the info.